src/server/game/Spells/SpellInfo.h

```diff
@@ -198,8 +198,9 @@ enum SpellCustomAttributes
     SPELL_ATTR0_CU_NEEDS_AMMO_DATA               = 0x00080000,
     SPELL_ATTR0_CU_BINARY_SPELL                  = 0x00100000,
     SPELL_ATTR0_CU_SCHOOLMASK_NORMAL_WITH_MAGIC  = 0x00200000,
-    SPELL_ATTR0_CU_LIQUID_AURA                   = 0x00400000,
+    SPELL_ATTR0_CU_DEPRECATED_LIQUID_AURA        = 0x00400000, // DO NOT REUSE
```

Comment:
DO NOT REUSE was added to have no conflicts between branches, but now it will be unused in both branches and looks like can be renamed to unused

Reply:
No, DO NOT REUSE means DO NOT REUSE, ever - this saves everyone from trying to use old fixes and breaking things even further

Reply:
Oh, right